librz/bin/p/bin_mbn.c

```diff
@@ -22,105 +21,102 @@ typedef struct sbl_header {
 	ut32 cert_sz;
 } SblHeader;
 
-// TODO avoid globals
-static SblHeader sb = { 0 };
+static bool read_sbl_header(RzBuffer *b, SblHeader *sb) {
+	ut64 offset = 0;
+	if (!(rz_buf_read_le32_offset(b, &offset, &sb->load_index) &&
```

Comment:
Probably better to split this into two functions - one for reading like `read_sbl_header` and one for checking.